### PR TITLE
Account Menu in Header

### PR DESCRIPTION
**Pull request by fritzschoff**
Put the account selection menu in the header. According to the design
Bildschirm­foto 2022-12-13 um 14 29 54
Bildschirm­foto 2022-12-13 um 14 30 07
Bildschirm­foto 2022-12-13 um 14 32 36

### CODE REVIEW

File: v3/ui/src/layouts/Default/Header.tsx [hunk 1]
```diff
@@ -140,6 +153,69 @@ export default function Header() {
...
+                  variant="outline"
+                  rightIcon={<ChevronDownIcon />}
+                  w="100%"
+                  maxW="180px"
+                >
+                  {id
+                    ? `Account #${id
+                        .substring(0, 3)
+                        .concat('...')
+                        .concat(id.substring(id.length - 3))}`
```
[the remark this answers is not in the document]
Comment (bachstatter) [reply v3/ui/src/layouts/Default/Header.tsx:171]: Maybe even a copy function, although that might be tricky given that it already is a button

**Review by 0xjocke — COMMENTED** [review]
Just one comment otherwise it looks good

**Review by noisekit — APPROVED** [review]

File: v3/ui/src/layouts/Default/Header.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (fritzschoff) [reply v3/ui/src/layouts/Default/Header.tsx:171]: will do one!

[the remark this answers is not in the document]
Comment (fritzschoff) [reply v3/ui/src/layouts/Default/Header.tsx]: called prettyString

**Pull request merged**